src/gzip.rs

```diff
@@ -409,7 +409,9 @@ impl Header {
         if flags & F_COMMENT != 0 {
             this.comment = Some(read_cstring(&mut reader)?);
         }
-        if flags & F_HCRC != 0 {
+        // checksum verification is skipped during fuzzing
+        // so that random data from fuzzer can reach actually interesting code
+        if flags & F_HCRC != 0 && cfg!(not(fuzzing)) {
```

Comment:
I prefer more general flag name than `fuzzing` (e.g., `skip_checksum_verification`). Since the flag may be available for testing or debugging purposes other than fuzzing.

Reply:
> However, "fuzzing" is the flag that's automatically passed by all three Rust fuzzers (cargo-fuzz, afl.rs, honggfuzz-rs), and I rather have them to Just Work.

I see. If there is such a reason, the flag name "fuzzing" is okay.
Could you add comments in source code about "When the flag is set" and "Who sets the flag".